File: C#/ExtendedServerAPI/ZooZExtendedServerAPI.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ExtendedSeverAPI
{
    class ZooZExtendedServerAPI
    {
        private static readonly string PRODUCTION_URL = "https://app.zooz.com/";
        private static readonly string SANDBOX_URL = "https://sandbox.zooz.co/";

        public static string zoozDeveloperId { get; private set; }

        public static string zoozServerAPIKey { get; private set; }

        public static string zoozServer { get; private set; }


        /// <summary>
        ///  Initialize the ZooZExtendedServerAPI to enable communication with your account
        /// </summary>
        /// <param name="zooz_Developer_Id">
        /// Developer email as entered in ZooZ Developer portal. </param>
        /// <param name="zooz_Server_API_Key">
        /// ZooZ Server API Key assigned to you upon registration. Can be found on the "Account" section of the ZooZ Developer Portal. </param>
        /// <param name="is_Sandbox">
        /// True to use sandbox environment, false to use live environment. </param>
        public static void init(string zooz_Developer_Id, string zooz_Server_API_Key, bool is_Sandbox)
        {
            if ((zooz_Developer_Id == "") || (zooz_Server_API_Key == ""))
            {
                //throw error this is empty
            }

            ZooZExtendedServerAPI.zoozDeveloperId = zooz_Developer_Id;
            ZooZExtendedServerAPI.zoozServerAPIKey = zooz_Server_API_Key;

            if (is_Sandbox)
            {
                ZooZExtendedServerAPI.zoozServer = SANDBOX_URL;
            }
            else
            {
                ZooZExtendedServerAPI.zoozServer = PRODUCTION_URL;
            }

        }
    }
}

File: Java/src/main/java/com/zooz/extended/java/lib/ZooZExtendedServerAPI.java
package com.zooz.extended.java.lib;

import com.zooz.extended.java.lib.exception.ZooZException;
import com.zooz.extended.java.lib.utils.ZooZStringUtils;

/**
 * Developer: Roy Keynan
 */
public class ZooZExtendedServerAPI {

    private static String zoozDeveloperId;
    private static String zoozServerAPIKey;
    private static String zoozServer;

    /**
     * Initialize the ZooZExtendedServerAPI to enable communication with your account
     *
     * @param zoozDeveloperId  - Developer email as entered in ZooZ Developer portal.
     * @param zoozServerAPIKey - ZooZ Server API Key assigned to you upon registration. Can be found on the "Account" section of the ZooZ Developer Portal.
     * @param zoozServer       - ZooZ Server which we want to connect to (PRODUCTION , SANDBOX)
     */
    public static void init(String zoozDeveloperId, String zoozServerAPIKey, String zoozServer) throws ZooZException {

        if (ZooZStringUtils.isEmpty(zoozDeveloperId)) {
            throw new ZooZException("Please enter your developer email as registered in ZooZ's developer portal");
        }

        if (ZooZStringUtils.isEmpty(zoozServerAPIKey)) {
            throw new ZooZException("Please enter ZooZ Server API Key as diaplayed in ZooZ's developer portal under \'My Account\'");
        }

        ZooZExtendedServerAPI.zoozDeveloperId = zoozDeveloperId;
        ZooZExtendedServerAPI.zoozServerAPIKey = zoozServerAPIKey;
        ZooZExtendedServerAPI.zoozServer = zoozServer;
    }

    /**
     * Get getZoozDeveloperId
     *
     * @return zoozDeveloperId
     */
    public static String getZoozDeveloperId() {
        return zoozDeveloperId;
    }

    /**
     * Get zoozServerAPIKey
     *
     * @return zoozServerAPIKey
     */
    public static String getZoozServerAPIKey() {
        return zoozServerAPIKey;
    }

    /**
     * Get zoozServer
     *
     * @return zoozServer
     */
    public static String getZoozServer() {
        return zoozServer;
    }

}
